releasenotes/notes/fixed-bug-in-function-call-in-diagnose-145c0abf0a705afd.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fixed a bug in diagnose core_endpoint.go where parameter didn't
+    include a ``.``.
```

Comment:
The releasenotes are aimed at users. We should describe more what feature is fixed rather than the internals.